scripts/check_connectivity.py

```
@@ -29,50 +26,65 @@ async def check_connectivity() -> None:
+                success = False
+        elif settings.graph_enabled and not graph:
+            print("⚠️ Graph enabled but no adapter created (check logs).")
             success = False
 
+        # Verify Cache
+        if settings.cache_backend == "redis" and cache:
+            try:
+                await cache.set("chronos_check", "ok", ttl=10)
+                val = await cache.get("chronos_check")
+                print(f"✅ Cache (redis) connected! Check value: {val}")
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)